### PR TITLE
FFWEB-2216: change message content

### PR DESCRIPTION
**Pull request by paulfcdd**
- Description: 
Change message content 

- Tested with Magento editions/versions: 
2.4.2

- Tested with PHP versions: 
7.4.20

### CODE REVIEW

**Comment by paulfcdd** [comment]
I've met a very strange problem:
when something is wrong (for example, FTP config) and file can not be uploaded or imported, the message is shows up, but when everything works fine (create file, upload to FTP, push import) the message doesn't show's up. The logic of the show/hide popup is keeped in file `src/view/adminhtml/web/js/ajax-button.js` and this is weird that Generate feed button didn't trigger function to show popup. I can't find the reason of such behavior, maybe you can @a-laurowski ?

**Comment by paulfcdd** [comment]
@a-laurowski This is the error which I talking about. It occures only if ftp upload end push import was successfully. 
photo_2021-12-13_14-32-21

If something failed the modal window looks like on screen below
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
 # Changelog
-## Unreleased
+## [v3.1.0] - 2021.12.03
```
Comment (a-laurowski) [inline CHANGELOG.md:2]: why this file is contained in the PR?
It's from branch which is already merged in master. This applies to the other files as the image file or web components scripts.
I guess this PR should contain changes in only 3 files so it must be rebased wrongly
image

why these commits from master are in between of the commits of this branch?

**Comment by a-laurowski** [comment]
> @a-laurowski This is the error which I talking about. It occures only if ftp upload end push import was successfully. photo_2021-12-13_14-32-21
> 
> If something failed the modal window looks like on screen below image


Is there anything wrong with the FTP upload? From message i assume, it is. I ran the export and if everything went fine, then i have this messages 
image

Which i have one problem. There is no need to inform customer that push import is started. Instead, i would like to see that import has finished successfully

When upload fails i have this, which looks the same as yours (except there is no system exception message)
image

Speaking of this system exception message (the red bar) this is impossible for that to be rendered in a response of ajax request. This could be a remnant of some older operation, like saving config

**Pull request closed without merging**